cluster/slack-moderator-words/deployment.yaml

```diff
@@ -0,0 +1,50 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: slack-moderator-words
+  labels:
+    app: slack-moderator-words
+spec:
+  replicas: 2
+  selector:
+    matchLabels:
+      app: slack-moderator-words
+  template:
+    metadata:
+      labels:
+        app: slack-moderator-words
+    spec:
+      containers:
+      - name: slack-moderator-words
+        image: gcr.io/kubernetes-tools/slack-moderator-words:latest
```

Comment:
version this or add imagePullPolicy: always?

Reply:
I forgot always is implicitly/magically the default for containers with image `:latest`, this is probably my least favorite thing in the API :|

Reply:
I think after we merge we will need to update this right? because it might trigger a job to build the images or not?